model/src/main/java/com/javahelp/model/user/ProviderUserInfo.java

```diff
@@ -8,6 +10,7 @@ public class ProviderUserInfo extends UserInfo {
     private boolean certified;
     private String address;
     private String phoneNumber;
+    private HashMap<String, Integer> attributeMap;
```

Comment:
Same as with `ClientUserInfo`, any information that goes here should probably be a field in the class.